### PR TITLE
Documentation fixes in INSTALL.md

### PR DESCRIPTION
**Pull request by bjoernfan**
Minor documentation fixes.

### CODE REVIEW

**Review by sspreitzer — APPROVED** [review]
lgtm

**Review by sspreitzer — APPROVED** [review]

File: INSTALL.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ This document provides a guide for intalling AWX.
   - [Start the build](#start-the-build)
   - [Post build](#post-build)
   - [Accessing AWX](#accessing-awx)
-- [Docker](#Docker)
-  - [Prerequisites](#prerequisites-1)
+- [Docker](#docker)
+  - [Prerequisites](#prerequisites-2)
```
Comment (jladdjr) [inline INSTALL.md:23]: Based on patterns used elsewhere in document, looks like this should still be `#prerequisites-1`

**Review by jladdjr — DISMISSED** [review]
minor issue with anchor

File: INSTALL.md (hunk 1 above)
Comment (bjoernfan) [reply INSTALL.md:23]: Yeah, #prerequisites-1 takes you to the Prerequisites heading under the OpenShift section, not the one in the Docker section. The other links I think you're thinking of, the ones with -1, are unique to the Docker section of the document.

**Review by wwitzel3 — APPROVED** [review]

**Comment by bjoernfan** [comment]
Thanks!

**Comment by wenottingham** [comment]
Thanks for the contribution!

**Pull request merged**